data/map planets.txt

```diff
@@ -1607,6 +1623,19 @@ planet Furnace
 		threshold 3500
 		fleet "Large Syndicate" 7
 
+planet Gagarin
+	attributes CCOR factory urban
+	landscape land/city0
+	description `Gagarin is the most populated planet in the region of what used to be the Confederated Councils of the Outer Rim, and was the first to be settled. All the settlements of Gagarin are connected to a planetary transport and infrastructure grid... although the grid hasn't been able to keep pace with demand from a growing population. Despite having been designed to be expanded with as demand increased, such resource-intensive infrastructure projects became harder and harder to maintain, due to shortages in supplies.`
```

Comment:
```suggestion
	description `Gagarin is the most populated planet in what used to be the Confederated Councils of the Outer Rim, and was the first to be settled in this region.`
	description `	All the settlements of Gagarin are connected to a planetary transport and infrastructure grid... although the grid hasn't been able to keep pace with Gagarin's growing population. Despite having been designed to be expanded as demand increased, such a resource-intensive infrastructure project became harder and harder to maintain, due to shortages in supplies. The collapse of the CCOR has only exacerbated the grid's deficiencies.`
```

Reply:
Not sure about this setup - are there other planets in the game with a one-liner and a more descriptive second paragraph?

Reply:
There are (e.g. New Austria) but I think that those one-liners are probably better to be avoided. I'll change it.

Reply:
Seems better, though the first line still reads odd to me. "This planet is in what used to be (Government), first settled in this region."